sys/drv/rtl8139.c

```diff
@@ -7,6 +7,7 @@
 #include <sys/libkern.h>
 #include <sys/devfs.h>
 #include <sys/devclass.h>
+#include <dev/rtl8139_reg.h>
```

Comment:
The `rtl8139_state` structure absolutely can't contain an `intr_handler_t` structure. Please note that interrupt handlers are only created inside the interrupt module and are allocated form the `M_INTR` kmalloc pool. Please remove this field. If you ever need an irq, you should use the `resource_t` structure.